newIDE/app/src/AssetStore/ExtensionStore/ExtensionInstallDialog.js

```diff
@@ -182,14 +182,16 @@ const ExtensionInstallDialog = ({
               <Trans>Version {' ' + extensionShortHeader.version}</Trans>
             </Text>
             <Line>
-              {extensionShortHeader.authors &&
-                extensionShortHeader.authors.map(author => (
-                  <UserPublicProfileChip
-                    user={author}
-                    key={author.id}
-                    isClickable
-                  />
-                ))}
+              <div style={{ flexWrap: 'wrap' }}>
```

Comment:
should this be inside the UserPublicProfileChip? 
I'm afraid that next time we use it we'll forget that wrapper

Reply:
Or a whole component?

Reply:
I was thinking of adding difficulty and size as chip on the begging of this line to save vertical space.